test/test_public_bindings.py

```diff
@@ -54,6 +54,7 @@ def test_no_new_bindings(self):
             "ConcreteModuleType",
             "ConcreteModuleTypeBuilder",
             "CONV_BN_FUSION",
+            "ConvBackend",
```

Comment:
You most likely don't want this right?

Reply:
should I prepend an underscore then to pass the test? `_ConvBackend`

Reply:
Yes I think so. Unless you want to make this public API and document it? :D